app/src/routes/$id.tsx

```diff
@@ -24,6 +36,10 @@ export const Route = createFileRoute("/$id")({
       hentEksisterendeInntektsmeldinger(params.id),
     ]);
 
+    if (true) {
+      throw new Error(FEILKODER.OPPGAVE_ER_UTGÅTT);
```

Comment:
Jeg er ikke helt sikker på om dette er rett pattern eller ikke. Står mellom dette, eller ha en egen route for utgått siden som vi redirecter til hvis status er utgått. Tanker?